lib/livebook_cli/server.ex

```diff
@@ -279,7 +279,7 @@ defmodule LivebookCLI.Server do
   defp append_path(url, path) do
     url
     |> URI.parse()
-    |> Map.update!(:path, &((&1 || "") <> path))
+    |> Map.update!(:path, &Path.join(&1 || "/", path))
```

Comment:
Looking at the use cases, we always want to override the path, so let's rename it to `set_path` and simplify to:

```suggestion
    |> Map.update!(:path, path)
```

wdyt?